elasticsearch/sgconfig/sg_config.yml

```diff
@@ -13,6 +13,21 @@ searchguard:
         http_authenticator:
           challenge: false
           type: io.fabric8.elasticsearch.plugin.auth.OpenShiftTokenAuthentication
+          config:
+             subjectAccessReviews:
+               prometheus:
+                 verb: get
+                 resource: /metrics
+               jaeger:
+                 verb: get
+                 resource: jaeger
+                 namespace: set-in-run-script
+                 resourceAPIGroup: elasticsearch.jaegertracing.io
+               elasticsearch-operator:
```

Comment:
shouldn't this already be in here? is a rebase needed?

Reply:
It exists for the authentication_backend but not the http_authenticator.  The class is both but is instantiated multiple times